cmd/workflow-trigger/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+
+	"github.com/codingpop/workflow-trigger"
+)
+
+var (
+	owner       string
+	repo        string
+	accessToken string
+	eventType   string
+)
+
+func init() {
+	flag.StringVar(&owner, "owner", "", "Repository owner")
+	flag.StringVar(&repo, "repo", "", "GitHub repository name")
+	flag.StringVar(&accessToken, "access-token", "", "Personal access token")
+	flag.StringVar(&eventType, "event-type", "", "A custom webhook event name")
+}
```

Comment:
These don't have to be in an `init`, normally they're just declared directly as global variables.
```suggestion
var (
	owner       = flag.String("owner", "", "Repository owner")
	repo        = flag.String("repo", "", "GitHub repository name")
      accessToken = flag.String("access-token", "", "Personal access token")
	eventType   = flag.String("event-type", "", "A custom webhook event name")
)
```